receiver/hostmetricsreceiver/internal/scraper/networkscraper/factory.go

```diff
@@ -36,7 +36,7 @@ type Factory struct {
 
 // CreateDefaultConfig creates the default configuration for the Scraper.
 func (f *Factory) CreateDefaultConfig() internal.Config {
-	return &Config{}
+	return &Config{ScrapeConntrack: false}
```

Comment:
We can/should rely on the zero value for booleans being `false, so this should also work 
```suggestion
	return &Config{}
```